### PR TITLE
📝 i18n Support

### PR DESCRIPTION
**Pull request by Cs4K1Sr4C**
-🌐 Added translations

-🌟 Improved the quality of already existing prompts.

-🆕 Introducing two new prompts:
  1. "Agent Prompt Creator" prompt for generate prompt which creates an Agent
  2. "Agent Creator" prompt based on its build-up prompt

-🆕 Added an input field to which allows the user to set the Agent's language.
  (Default agent's language is the browser's detected language; the language is changeable by entering the desired language and it's going to be processed in each prompt)

- Security improvements:
  1. CSRFToken
  2. Session Fixation

### CODE REVIEW

**Comment by awtkns** [comment]
@Cs4K1Sr4C seems to still be failing, running ` npm i --save-dev @types/cookie` seems to have fixed it for me

**Comment by awtkns** [comment]
@Cs4K1Sr4C you may want to try running `npm run build` locally. It seems it is now failing with: ```

Type error: Argument of type '{ httpOnly: boolean; sameSite: string; maxAge: number; path: string; }' is not assignable to parameter of type 'CookieSerializeOptions'.
--
09:51:06.700 | Types of property 'sameSite' are incompatible.
09:51:06.700 | Type 'string' is not assignable to type 'boolean \| "strict" \| "lax" \| "none" \| undefined'.

```

File: src/hooks/useAuth.ts [hunk 1]
```diff
@@ -26,10 +27,18 @@ export function useAuth(): Auth {
...
+
+    // Fetch CSRF token
+    /*fetch("/api/csrf-token")
+      .then((res) => res.json())
+      .then((data) => setCsrfToken(data.csrfToken))
+      .catch(() => setCsrfToken(""));
       .catch(() => undefined);
-  }, [session, status]);
+  }, [session, status]);*/
+})
```
Comment (awtkns) [inline src/hooks/useAuth.ts:38]: These changes should probably be reverted.

**Comment by asim-shrestha** [comment]
Going to close this given we have the other PR merged and one more PR open for the prompt side translations

**Pull request closed without merging**